paddlecloud/paddlecloud/settings.py

```diff
@@ -323,7 +323,7 @@
 ETCD_IMAGE="quay.io/coreos/etcd:v3.2.1"
 
 # domains that allow notebook to enter
-NOTEBOOK_DOMAINS=["www.paddlepaddle.org"]
+NOTEBOOK_DOMAINS=["cloud.paddlepaddle.org"]
```

Comment:
Why this? Can it be a template or sample string?

Reply:
Oh, it's just used for stay the same with `ALLOWED_HOSTS` , but I saw there is already some discuss about the template, I will remove this change..